test/fops/file.c

```diff
@@ -53,6 +55,17 @@ obj_bulk(void)
 		} else if (ret != ENOENT && ret != EBUSY && ret != EINVAL)
 			testutil_die(ret, "session.open_cursor bulk");
 	}
+
+	if (explicit_transaction) {
```

Comment:
I suggest adding a comment here and in `drop` about why the handling of `EINVAL` is special and different here than in the other functions.